src/content/config.ts

```diff
@@ -16,4 +17,5 @@ const guidesCollection = defineCollection({
 export const collections = {
   'guides': guidesCollection,
   'connections': guidesCollection,
+  'api': guidesCollection
```

Comment:
✏️ Suggestion: Maybe we should rename this variable to something more generic like `collection`.